src/LabLog/Migrations/20170928082432_InitialCreate.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace LabLog.Migrations
+{
+    public partial class InitialCreate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "LabEvents",
+                columns: table => new
+                {
+                    Version = table.Column<int>(type: "INTEGER", nullable: false)
+                        .Annotation("Sqlite:Autoincrement", true),
+                    EventBody = table.Column<string>(type: "TEXT", nullable: true),
+                    EventType = table.Column<string>(type: "TEXT", nullable: true),
+                    RoomId = table.Column<Guid>(type: "BLOB", nullable: false),
```

Comment:
type blob? Really?

Reply:
No idea, automatic.

Reply:
Guid is binary data, would you need to change the datatype of RoomId in LabEvent.cs to string, and then generate a Guid and stringify it to store in RoomId?

Reply:
Or is it the done thing to simply modify the migration files created by ef migrations?

Reply:
https://stackoverflow.com/questions/18954130/can-we-use-guid-as-a-primary-key-in-sqlite-database

Reply:
Oooh right. I'm used to using MsSql. Yes this all makes more sense now